libs/net/dal/Migrations/Initial/Up/PostUp/02-Ingest.sql

```diff
@@ -144,151 +147,11 @@ INSERT INTO public.ingest (
   , ''
   , DEFAULT_USER_ID
   , ''
-), (
-  'Globe and Mail'
-  , '' -- description
-  , true -- is_enabled
-  , paperId -- media_type_id
-  , (SELECT id FROM public.source WHERE code = 'GLOBE') -- source_id
-  , 'GLOBE' -- topic
-  , weeklyPrintId -- product_id
-  , '{ "timeZone": "Pacific Standard Time",
-      "language": "en-CA",
-      "post": true,
-      "import": true,
-      "papername":"pubdata!name",
-      "headline":"hl1",
-      "summary":"hl1",
-      "story":"body.content",
-      "author":"byline",
-      "date":"pubdata!date.publication",
-      "id":"pubdata!id",
-      "section":"pubdata!position.section",
-      "page":"pubdata!position.sequence",
-      "item":"nitf","dateFmt":"yyyyMMdd",
-      "selfPublished": true,
-      "dateOffset": -1 }' -- configuration
-  , 0 -- schedule_type
-  , 3 -- retry_limit
-  , 5 -- source_connection_id
-  , 6 -- destination_connection_id
```

Comment:
Hardcoded primary keys that are generated result in issues more often than not.  I've replaced them with a variable below.